cardano-testnet/src/Testnet/Start/Types.hs

```diff
@@ -33,9 +33,9 @@ data CardanoTestnetOptions = CardanoTestnetOptions
   { -- | List of node options. Each option will result in a single node being
     -- created.
     cardanoNodes :: [TestnetNodeOptions]
-  , cardanoNodeEra :: AnyCardanoEra
-  , cardanoEpochLength :: Int
-  , cardanoSlotLength :: Double
+  , cardanoNodeEra :: AnyCardanoEra -- ^ The era to start at
+  , cardanoEpochLength :: Int -- ^ Epoch length, in milliseconds
```

Comment:
The epoch length is measured in slots. We need to update the parser help as well.

Reply:
Added a commit updating the parser :+1: